stock_production_lot_traceability/models/stock_move_line.py

```diff
@@ -11,7 +11,7 @@ class StockMoveLine(models.Model):
     def write(self, vals):
         # OVERRIDE to invalidate the cache of the lots' produce/consume fields,
         # as they're computed from the move lines that consume/produce them.
-        self.env["stock.production.lot"].invalidate_cache(
+        self.env["stock.lot"].invalidate_cache(
```

Comment:
Please change all invalidate_cache() to invalidate_model()

See: https://github.com/OCA/stock-logistics-workflow/actions/runs/6559573993/job/17815435386?pr=1403#step:8:214

Reply:
@rousseldenis , I just updated the code, it would be nice if you could help to review it again. Thank you